src/PlatformBenchmarks/BenchmarkApplication.cs

```diff
@@ -90,17 +117,39 @@ private static void PlainText(PipeWriter pipeWriter)
             writer.Write(DateHeader.HeaderBytes);
 
             // Content-Type header
-            writer.Write(_headerContentTypeText);
+            writer.Write(_headerContentTypeJson);
```

Comment:
Would stackalloc'ing the object here be of any benefit?

Reply:
Which object?

Reply:
Bleh, wrong line. The obejct that gets serialized to JSON.